### ISSUE TITLE
Report  Command failed: /usr/local/bin/swiftlint

### ISSUE THREAD

**Issue by FrizzleFur** [vknabel/vscode-swiftlint#29]
```
/usr/local/bin/swiftlint --quiet --reporter jsonerror opening input file '/Users/mac/Documents/.swift' (No such file or directory)Most rules will be skipped because sourcekitd has failed.Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=260 "The file “.swift” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Users/mac/Documents/../.swift, NSUnderlyingError=0x7ff61d0f84a0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}: file /private/tmp/swiftlint-20200404-90819-1vnlirh/.build/checkouts/SourceKitten/Source/SourceKittenFramework/File.swift, line 28
```

```
Error: Command failed: /usr/local/bin/swiftlint --quiet --reporter json
error opening input file '/Users/mac/Documents/.swift' (No such file or directory)
Most rules will be skipped because sourcekitd has failed.
Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=260 "The file “FeedSwitchUserGuideView.swift” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Users/mac/Documents/development/LarkProjects/ios-infra/Bizs/LarkGuide/LarkGuide/view/FeedSwitchUserGuideView.swift, NSUnderlyingError=0x7ff61d0f84a0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}: file /private/tmp/swiftlint-20200404-90819-1vnlirh/.build/checkouts/SourceKitten/Source/SourceKittenFramework/File.swift, line 28

	at ChildProcess.exithandler (child_process.js:312:12)
	at ChildProcess.emit (events.js:315:20)
	at maybeClose (internal/child_process.js:1021:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
os: darwin
```

**Comment by vknabel** [comment]
Hey @FrizzleFur 

Thanks for your report. Do you have some additional information when this error happened?
Was your swiftlint up to date? Did you use vscode share or is there anything else special about your setup?

**Comment by FrizzleFur** [comment]
> Hey @FrizzleFur
> 
> Thanks for your report. Do you have some additional information when this error happened?
> Was your swiftlint up to date? Did you use vscode share or is there anything else special about your setup?

its OK
